core/src/main/scala/za/co/absa/spline/harvester/builder/SourceUri.scala

```diff
@@ -24,6 +24,8 @@ object SourceUri {
 
   def forJDBC(connectionUrl: String, table: String): String = s"$connectionUrl:$table"
 
+  def forCassandra(keyspace: String, table: String): String = s"$keyspace:$table"
```

Comment:
```suggestion
  def forCassandra(keyspace: String, table: String): String = s"cassandra:$keyspace:$table"
```
A source URI has to be unique, as it is used to link multiple execution plans into a high-level end-to-end data lineage.
`"$keyspace:$table"` uniquely identifies a Cassandra table, but it might not to be unique source identifier for Spline. Imagine another job reads from a MongoDB, where a database and a collection happened to have the same names as a keyspace and a table in Cassandra. If that happens Spline would treat them as the same source and display lineage incorrectly.  

To prevent this we always prefix URIs with a scheme matching the source type, e.g. `file:`, `jdbc:`, `kafka:`, `excel:` etc.